x-pack/distributions/internal/observabilitySRE/qa/smoke/docker/certs/generate.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
nit: add the generated pki bits to the `.gitignore`

~~~
printf '%s\n' '*.crt' '*.csr' '*.key' > x-pack/distributions/internal/observabilitySRE/qa/smoke/docker/certs/.gitignore
~~~